crates/flipperzero/src/notification/mod.rs

```diff
@@ -0,0 +1,308 @@
+//! Furi notifications.
+
+use core::ffi::c_char;
+
+use flipperzero_sys as sys;
+use flipperzero_sys::furi::UnsafeRecord;
+
+///Default notification messages.
+pub mod messages;
+///Default notification notes.
+pub mod notes;
+///Default notification sequences.
+pub mod sequences;
+
+const RECORD_NOTIFICATION: *const c_char = sys::c_string!("notification");
+
+/// A handle to the Notification app.
```

Comment:
I find `NotificationApp` a bit confusing, even if its how the handle is called in the SDK. `NotificationService` would probably be a better name (this is even [how the SDK documentation refers to it](https://github.com/flipperdevices/flipperzero-firmware/blob/663eb6c/applications/ReadMe.md#services)).